.github/workflows/claude-review.yml

```
@@ -67,4 +67,4 @@ jobs:
           use_sticky_comment: true
           
           # Allow Claude to run tests and linting
-          allowed_tools: "Bash(npm run test),Bash(npm run lint),Bash(npm run typecheck),Bash(npm run build)"
\ No newline at end of file
+          allowed_tools: "Bash(npm run test),Bash(npm run lint),Bash(npm run lint:fix),Bash(npm run check),Bash(npm run check:fix),Bash(npm run format),Bash(npm run typecheck),Bash(npm run build)"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)